hytest.hy

```diff
@@ -191,11 +191,14 @@
 (def mods [])
 (def tests (OrderedDict))
 
+(defn source-file []
+  (and (in "__file__" (globals)) __file__))
```

Comment:
Two problems here:

- Since this is evaluated inside `hytest.hy`, the return value will *always* be the path to `hytest.hy`...even though it's supposed to be the test module path. (You can just stick a backtick in front of the `(and` to quote it, and it should work correctly.)

- I think it would be a bit better if this defaulted to something like `"<unknown-file>"` or similar. As-is, if `__file__` is undefined, it'll end up putting `False` as the file name...which seems a bit odd to me.